src/context/engagement_context/__tests__/engagement_context.spec.tsx

```diff
@@ -117,6 +117,17 @@ describe('Engagement Context', () => {
     });
   });
 
+  test('Can delete an engagement', async () => {
+    await act(async () => {
+      const { result, waitForNextUpdate } = getHook();
+      await waitForNextUpdate;
+      const engagement = Engagement.fromFake();
+      result.current.deleteEngagement(engagement);
+      await waitForNextUpdate();
+      expect(result.current.engagements?.length).toEqual(0);
```

Comment:
This assertion is asserting something that was never modified in the test. By default, `engagements.length` equals 0, and nothing in this test modifies the original `engagements` array.
In order to make this test assert correctly, you will need to have engagements in the engagements array, then delete one of those engagements.